napari/layers/base/base.py

```diff
@@ -1182,33 +1187,32 @@ def _slice_dims(
 
     def _make_slice_input(
         self,
-        dims: Dims = None,
+        dims: Dims,
     ) -> _SliceInput:
-        world_ndim = self.ndim if dims is None else dims.ndim
+        world_ndim: int = self.ndim if dims is None else dims.ndim
         if dims is None:
             # if no dims is given, "world" has same dimensionality of self
             # this happens for example if a layer is not in a viewer
             # in this case, we assume all dims are displayed dimensions
             world_slice = _ThickNDSlice.make_full((np.nan,) * self.ndim)
         else:
             world_slice = _ThickNDSlice.from_dims(dims)
-
-        order = (
-            tuple(range(world_ndim))
+        order_1 = (
```

Comment:
maybe use name `order_array` and `order`? 
it may be more readable than numbers.

Reply:
Done.